python/docs/source/user_guide/index.rst

```diff
@@ -24,4 +24,5 @@ User Guide
     :maxdepth: 2
 
     arrow_pandas
+    python_packaging
```

Comment:
No big deal but I would rename the file `python_packaging` to `python_packages`.

Reply:
OK. I will change it.